tools/data/build_audio_features.py

```diff
@@ -0,0 +1,265 @@
+import argparse
+import glob
+import os
+import os.path as osp
+import sys
+from multiprocessing import Pool
+
+import numpy as np
+from scipy.io import wavfile
+
+try:
+    import librosa
+    import lws
+    from tqdm import tqdm
+except ImportError:
+    print('Please import librosa, lws and tqdm first.')
+
+sys.path.append('..')
+
+
+class AudioTools:
+    """All methods related to audio feature extraction.
+
+    Args:
+        frame_rate (int): The frame rate per second of the video. Default: 30.
+        sample_rate (int): The sample rate for audio sampling. Default: 16000.
+        num_mels (int): Number of channels of the melspectrogram. Default: 80.
+        fft_size (int): fft_size / sample_rate is window size. Default: 1280.
+        hop_size (int): hop_size / sample_rate is step size. Default: 320.
+    """
+
+    def __init__(self,
+                 frame_rate=30,
+                 sample_rate=16000,
+                 num_mels=80,
+                 fft_size=1280,
+                 hop_size=320):
+        self.frame_rate = frame_rate
+        self.sample_rate = sample_rate
+        self.silence_threshold = 2
+        self.num_mels = num_mels
+        self.fmin = 125
+        self.fmax = 7600
+        self.fft_size = fft_size
+        self.hop_size = hop_size
+        self.frame_shift_ms = None
+        self.min_level_db = -100
+        self.ref_level_db = 20
+        self.rescaling = True
+        self.rescaling_max = 0.999
+        self.allow_clipping_in_normalization = True
+        self.log_scale_min = -32.23619130191664
```

Comment:
since the args except `frame_rate`, `sample_rate`, `num_mels`, `fft_size` and `hop_size` are hardcoded, they can be set as constant variables whose variable name are capitalized.

Reply:
https://github.com/open-mmlab/mmaction2/blob/master/demo/webcam_demo.py#L14-L24